src/BuildingBlocks/Data/Masa.BuildingBlocks.Data.Contracts/Internal/ServiceScoped.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+[assembly: InternalsVisibleTo("Masa.BuildingBlocks.Data")]
+// ReSharper disable once CheckNamespace
+
+namespace Masa.BuildingBlocks.Data;
+
+public class ServiceScoped
```

Comment:
ServiceXXX? XXXService?

Reply:
The currently used format is: ServiceXXX